bundler/lib/bundler/man/bundle-cache.1.ronn

```diff
@@ -4,6 +4,7 @@ bundle-cache(1) -- Package your needed `.gem` files into your application
 ## SYNOPSIS
 
 `bundle cache`
+`bundle package`
```

Comment:
I think we need to add another break:

```suggestion
`bundle cache`

`bundle package`
```

Otherwise we will get a single line with `bundle cache bundle package` in man.

Reply:
Done in 6080e96